### ISSUE TITLE
Add wrap-up table for ensemble methods

### ISSUE THREAD

**Issue by ArturoAmorQ** [INRIA/scikit-learn-mooc#448]
Adding a wrap-up table to summarize the differences and similarities between bagging vs. boosting methods (ways of training, combining and computation times) may help setting down ideas and improve the success rate of Quiz M6.3 Q1 (which is currently below 70%).

It could be at the end of the _Ensemble based on boosting_ lectures, i.e., inside the [ensemble_hist_gradient_boosting notebook](https://inria.github.io/scikit-learn-mooc/python_scripts/ensemble_hist_gradient_boosting.html).

What do you think?

**Comment by lesteve** [comment]
There is this table for RandomForest vs Bagging https://inria.github.io/scikit-learn-mooc/python_scripts/ensemble_random_forest.html but we don't have a table for comparing bagging and boosting.

**Comment by ogrisel** [comment]
I think it's a good idea.

**Comment by ogrisel** [comment]
We could have a similar table at the end of this notebook:

https://inria.github.io/scikit-learn-mooc/python_scripts/ensemble_hyperparameters.html

or in a new notebook (without any code) right after this one.

**Comment by ArturoAmorQ** [comment]
We have such table in the ["Intuitions on ensemble models: boosting" slides](https://inria.github.io/scikit-learn-mooc/ensemble/boosting_slides.html), which were introduced in https://github.com/INRIA/scikit-learn-mooc/pull/471.

**Comment by ogrisel** [comment]
> We have such table in the "Intuitions on ensemble models: boosting" slides, which were introduced in #471.
> Should we still add it in a notebook?

Only if we expand it a bit, for instance by including extra info about the influence of important hyper-parameters, e.g.:

- too many trees can cause overfitting in gradient boosting but not for random forests.

- Gradient boosting requires tuning a learning rate parameter while random forests do no have such a parameter.
